pallets/ddc-verification/Cargo.toml

```diff
@@ -20,6 +20,7 @@ frame-support = { workspace = true }
 frame-system = { workspace = true }
 itertools = { workspace = true }
 log = { workspace = true }
+pallet-staking = { workspace = true }
```

Comment:
ddc-payouts also need to change as it won't work, as caller is OCW and not validator. so it needs to be T::ValidatorVisitor::is_OCW_validator(caller). as we don't want to explicitly give access to the mapping to the payouts pallet so the verification needs to be factored inside the method

			let caller = ensure_signed(origin)?;
			ensure!(
				T::ValidatorVisitor::get_active_validators().contains(&caller),
				Error::<T>::Unauthorised
			);